src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -104,37 +104,13 @@
                     (aset xhr (name k) v))
                   xhr))
           call-on-done (fn []
-                         (let [status-code (.-status xhr)
-                               orig-response {:xhr xhr
-                                              :status-code status-code
-                                              :success? (< 199 status-code 300)
-                                              :status-text (.-statusText xhr)
-                                              :get-parsed-response #(parse-json-string (.-responseText xhr))}]
-                           (if (= status-code 401)
-                             (let [us-xhr (js/XMLHttpRequest.)]
-                               (.addEventListener us-xhr "loadend"
-                                 (fn []
-                                   (let [us-status (.-status us-xhr)
-                                         parsed-us-response (parse-json-string (.-responseText us-xhr))
-                                         keywordized-us-response (keywordize-keys parsed-us-response)]
-                                     (cond
-                                       ;TODO: Fix this with a real log-out once the login bug is fixed and logout is implemented.
-                                       (= us-status 401) (do (delete-access-token-cookie)
```

Comment:
does this deletion mean we realistically never call delete-access-token-cookie any more? After this PR, it looks like the only place we call it any more is in the else clause of set-access-token-cookie, which is an edge case.